src/rust/engine/options/src/args.rs

```diff
@@ -81,18 +81,6 @@ impl Arg {
     fn matches_negation(&self, id: &OptionId) -> bool {
         self._matches(id, true)
     }
-
-    fn to_bool(&self) -> Result<Option<bool>, ParseError> {
```

Comment:
It was either pass a FromfileExpander into this method or move the method. The latter seemed like the better choice.